.github/workflows/test.yml

```diff
@@ -24,4 +35,10 @@ jobs:
       run: |
         cd scholar-example
         go build
-        go test
\ No newline at end of file
+        go test
+    - name: Run Integration Tests (with warnings only)
+      run: |
+        echo "Running integration tests against live Google Scholar API..."
+        echo "Note: These tests may fail due to rate limits or network issues - this is expected"
+        cd /home/runner/work/scholar/scholar
+        go test -tags integration -v || echo "⚠️  Integration tests failed (this is expected and does not break CI)"
```

Comment:
is it possible that we can have a warning show up in GH actions if this has a failure that we don't consider fatal?